src/sentry/integrations/github/integration.py

```diff
@@ -199,16 +202,15 @@ def get_pipeline_views(self):
         return [GitHubInstallationRedirect()]
 
     def get_installation_info(self, installation_id):
+        client = self.get_client()
         headers = {
             # TODO(jess): remove this whenever it's out of preview
             "Accept": "application/vnd.github.machine-man-preview+json",
         }
         headers.update(jwt.authorization_header(get_jwt()))
-        with http.build_session() as session:
-            resp = session.get(
-                f"https://api.github.com/app/installations/{installation_id}", headers=headers
-            )
-            resp.raise_for_status()
+        resp = client.request(
+            method="GET", headers=headers, path=f"/app/installations/{installation_id}34"
```

Comment:
That's an interesting 34 you have in there 👀

Reply:
ahhh i didnt even mean to open the PR this early but 'cmd + enter' already pinged everyone!! -- will remove that testing edit 👀